mypy/main.py

```diff
@@ -1561,7 +1572,8 @@ def install_types(
         print()
     print("Installing missing stub packages:")
     assert options.python_executable, "Python executable required to install types"
-    cmd = [options.python_executable, "-m", "pip", "install"] + packages
+    pip_options = ["--user"] if options.user else []
```

Comment:
If more pip options are supported in the future, this could go into a proper `derive_pip_options` function or similar